content/en/metrics/advanced-filtering.md

```diff
@@ -19,13 +19,15 @@ further_reading:
 
 ## Overview
 
-Whether you're using the Metrics Explorer, monitors, or dashboards to query metrics data, you can filter the data to narrow the scope of the timeseries returned. Any metric can be filtered by tag(s) using the **from dropdown** to the right of the metric. 
+When using the Metrics Explorer, monitors, or dashboards to query metrics data, you can filter the data to narrow the scope of the timeseries returned. Any metric can be filtered by tag(s) using the **from** field to the right of the metric. 
 
-{{< img src="metrics/advanced-filtering/tags.png" alt="Filter with tags" style="width:80%;" >}}
+You can also perform advanced filtering with Boolean or Wildcard tag value filters. For queries outside of metrics data such as logs, traces, Network Monitoring, Real User Monitoring, Synthetics, or Security, see the [Log Search Syntax][1] documentation for configuration.
 
-You can also perform advanced filtering with Boolean or Wildcard tag value filters. For queries outside of metrics data such as logs, traces, Network Monitoring, Real User Monitoring, Synthetics, or Security, see the [Log Search Syntax documentation][1] for configuration.
+To use the examples presented, copy and paste into your graph editor to see the the query in the UI by clicking on the code icon `</>`.
 
-### Boolean filtered queries 
+{{< img src="metrics/advanced-filtering/graph_editor_code_option.mp4" alt="Click code icon to see the raw query" video=true >}}
```

Comment:
I would recommend moving this to the Boolean filtered queries section, right before the first example.

Reply:
@estherk15 just checking if you plan on moving this or will leave here?

Reply:
Thanks @maycmlee , I moved it from the overview to the Boolean filtered query examples. I put it below the example to match up the format of the other images. If you think it would look better above let me know!

Reply:
>  I moved it from the overview to the Boolean filtered query examples.

Strange, I don't see the change: https://a.cl.ly/geuOQOJN

Reply:
Ooooh, I only moved the video. I'll update it!